src/css/app.general.scss

```diff
@@ -0,0 +1,16 @@
+body {
+  margin: 0;
```

Comment:
We have styles reset. We don't need these styles here. You can define only `font-family` here